modules/javafx.graphics/src/shims/java/com/sun/javafx/css/BitSetShim.java

```diff
@@ -73,4 +82,107 @@ public static int size(BitSet s) {
         return s.size();
     }
 
+    public static BitSetShim<PseudoClass> getPseudoClassInstance() {
+        return new BitSetShim<>(new PseudoClassState());
+    }
+
+    private final BitSet<T> delegate;
+
+    private BitSetShim(BitSet<T> delegate) {
+        this.delegate = delegate;
+    }
+
+    // Generated delegate methods:
```

Comment:
Very minor: these methods are not generated as part of the build process that would warrant pointing it out.

Reply:
Yes, it might be worth adding a comment to that effect.